Hunting Queries/SecurityEvent/Windows System Shutdown-Reboot(T1529)

```diff
@@ -0,0 +1,26 @@
+id: 024b3726-add7-4e06-842d-932034ba21f7
+name: Windows System Shutdown/Reboot(Sysmon)
+description: |
+  'This detection uses Sysmon telemetry to detect System Shutdown/Reboot (MITRE Technique: T1529)'
+
+requiredDataConnectors:
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvent
+tactics:
+  - Impact
+relevantTechniques:
+  - T1529
+query: |
+let timeframe = 1d;
+Event
+| where TimeGenerated >= ago(timeframe)
+//This query uses sysmon data depending on table name used this may need updataing
+| where Source == "Microsoft-Windows-Sysmon"
+| where (EventID == 1) 
+| where Image contains "shutdown.exe" 
```

Comment:
Sysmon Events are not parsed out since this query only calls the Event table and not the Parser.   There is a requirement to load one of the [Sysmon parsers](https://github.com/Azure/Azure-Sentinel/tree/master/Parsers/Sysmon) into the workspace before this will function, otherwise you have to do the parsing first.  Because Parsers are not automatically included, for this to work as a hunting query without any additional importing of parsers by a user, you will want to do your own parsing in the query.  This detection has a good example on how to do this - https://github.com/Azure/Azure-Sentinel/blob/master/Detections/SecurityEvent/ADFSKeyExportSysmon.yaml - and the parser itself can show you the syntax for all the Event ID 1 fields available.  Everything including Image and below will not resolve since it has not been parsed out.